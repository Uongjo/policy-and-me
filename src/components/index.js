import NavBar from './NavBar';
import Person from './Person';
import Legislator from './Legislator';
import Bill from './Bill';
import InterestGroup from './InterestGroup';
import Pagination from './Pagination';
import Search from './Search';
import VisualizationMap from './VisualizationMap';
import VisualizationHeatMap from './VisualizationHeatMap';
import VisualizationBarChart from './VisualizationBarChart';
import VisualizationBubbleChart from './VisualizationBubbleChart';
import DoctorMap from './DoctorMap';
import DoctorCityRating from './DoctorCityRating';
import DoctorBarChart from './DoctorBarChart'
import Loading from './Loading';

export {
  NavBar, 
  Person, 
  Legislator, 
  Bill, 
  InterestGroup, 
  Pagination, 
  Search, 
  VisualizationMap, 
  VisualizationHeatMap,
  VisualizationBarChart,
  VisualizationBubbleChart,
  DoctorMap,
  DoctorCityRating,
  DoctorBarChart,
  Loading,
};