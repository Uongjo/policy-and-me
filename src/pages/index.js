import Splash from './Splash';
import About from './About';
import Bills from './Bills';
import Legislators from './Legislators';
import InterestGroups from './InterestGroups';
import LegislatorInstance from './LegislatorInstance';
import BillInstance from './BillInstance';
import InterestGroupInstance from './InterestGroupInstance';
import GlobalSearch from './GlobalSearch';
import Visualization from './Visualization';

export {
  Splash, 
  About, 
  Bills, 
  Legislators, 
  InterestGroups, 
  LegislatorInstance, 
  BillInstance,
  InterestGroupInstance,
  GlobalSearch,
  Visualization,
};