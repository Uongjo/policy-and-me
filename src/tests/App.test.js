import React from "react"
import renderer from "react-test-renderer"
import {BrowserRouter} from "react-router-dom"
import {About, Splash, Bills, InterestGroups, Legislators} from "../pages"
import {Bill, InterestGroup, Legislator, Person, Pagination, NavBar, Search, highlightSearch} from "../components"
import {configure, shallow, mount} from "enzyme"
import Adapter from 'enzyme-adapter-react-16'
import {createMount} from '@material-ui/core/test-utils'



configure({ adapter: new Adapter() });

describe("Components did load normally", () => {

    const emptySearch = ""

    // Test 1: Check Bill component
    it("Bill loads normally", () => {
        // short_title, introduced_date, primary_subject, sponsor_name, committees
        const bill = {short_title: "Bill-1", introduced_date: "1999", primary_subject: "Saftey", sponsor_name: "Joe", committees: "ABC"}
        const tree = renderer.create(<Bill bill = {bill} searchValue = {emptySearch} />).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 2: Check InterestGroup component
    it("InterestGroup loads normally", () => {
        // name, address, city, phone, email, 
        const ig = {name:"Test", address: "123 Test Ave", city: "Somewhere", phone:"111-111-1111", email: "test@google.com"}
        const tree = renderer.create(<InterestGroup interestGroup = {ig} searchValue = {emptySearch}/>).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 3: Check Legislator component
    it("Legislator loads normally", () => {
        // first_name, last_name, title, party, gender, state
        const leg = {first_name:"First", last_name: "Last", title: "1st Class", party:"D", gender: "M", state: "TX"}
        const tree = renderer.create(<Legislator legislator = {leg} searchValue = {emptySearch}/>).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 4: Check person component
    it("Person loads normally", () => {
        const tree = renderer.create(<Person/>).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 5: check NavBar component
    it("NavBar loads normally", () =>{
        const tree = renderer.create(<BrowserRouter><NavBar/></BrowserRouter>).toJSON()
        expect(tree).toMatchSnapshot()
    })
    
    // Test 6: check pagination component
    it("Pagination loads normally", () =>{
        // setPageIndex, numPages
        const pagin = {setPageIndex: 0, numPages: 10}
        const tree = renderer.create(<Pagination {...pagin}/>).toJSON()
        expect(tree).toMatchSnapshot()
    })
})

describe("pages did load normally", () => {

    // Test 7: check about page 
    it("About did load normally", () => {
        const tree = renderer.create(<About/>).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 8: check bills page
    it("Bills did load normally", () => {
        const defaultProps = {
            match: { params: { id: undefined } }
        };
        const tree = renderer.create(<Bills {...defaultProps} />).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 9: check legislator page
    it("Legislators did load normally", () => {
        const defaultProps = {
            match: { params: { id: undefined } }
        };
        const tree = renderer.create(<Legislators {...defaultProps} />).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 10: check interestgroup page
    it("InterestGroups did load normally", () => {
        const defaultProps = {
            match: { params: { id: undefined } }
        };
        const tree = renderer.create(<InterestGroups {...defaultProps} />).toJSON()
        expect(tree).toMatchSnapshot()
    })

})

describe("Search, Sort, Filter", () => {

    const searchProps = {setModel: [],
        model: "sigs",
        setSearchValue: "",
        sortOptions: { name: "Name" },
        filterOptions: { state: "State" } ,
        setFilterKey: "",
        setFilterValue: "",
        setPageIndex: 0
    }

    // Test 11: check splash page
    it("Splash did load normally", () => {
        const tree = renderer.create(<BrowserRouter><Splash/></BrowserRouter>).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 12: check Search component
    it("Search loads normally", () =>{
        const tree = renderer.create(<BrowserRouter><Search {...searchProps}/></BrowserRouter>).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 13: check highlight
    it("Highlight did load normally", () => {
        const instance = {name: "bill", last: "clinton", state: "TX"}
        const searchValue = "bill"
        const tree = renderer.create(<highlightSearch instance = {instance} searchValue = {searchValue}/>).toJSON()
        expect(tree).toMatchSnapshot()
    })

    // Test 14: Check search loads initial state correctly
    it("Search loads props correctly", () =>{
        const wrapper = shallow(<Search {...searchProps}/>)
        const state = {
            sortValue: "",
            searchValue: "",
            filterKey: "",
            filterValue: "",
        }
        expect(wrapper.state()).toEqual(state)
    })

    // Test 15: check sort value is updated
    it("sort by is updated correctly", () =>{
        const wrapper = mount(<Search {...searchProps}/>)
        wrapper.find("select").at(0).simulate("change", {target: {value: "name"} })
        expect(wrapper.state().sortValue).toEqual("name")
    })

    // Test 16: Check filter value is updated
    it("filter by is updated correctly", () =>{
        const wrapper = mount(<Search {...searchProps}/>)
        wrapper.find("select").at(1).simulate("change", {target: {value: "state"} })
        expect(wrapper.state().filterKey).toEqual("state")
    })

    // Test 17: Check search value is updated
    it("Search value is updated correctly", () =>{
        const mount = createMount({dive: true})
        const wrapper = mount(<Search {...searchProps}/>)
        wrapper.find("input").at(0).simulate("change", {target: {value: "Adams"} })
        expect(wrapper.state().searchValue).toEqual("Adams")
    })

    // Test 18: Check filter value is updated
    it("filter value is updated correctly", () =>{
        const mount = createMount({dive: true})
        const wrapper = mount(<Search {...searchProps}/>)
        wrapper.find("input").at(1).simulate("change", {target: {value: "State"} })
        expect(wrapper.state().filterValue).toEqual("State")
    })

    // Test 19: Check filter by and filter value work together
    it("filter by and filter value update", () =>{
        const mount = createMount({dive: true})
        const wrapper = mount(<Search {...searchProps}/>)
        wrapper.find("select").at(1).simulate("change", {target: {value: "State"} })
        wrapper.find("input").at(1).simulate("change", {target: {value: "TX"} })
        expect(wrapper.state().filterValue).toEqual("TX")
        expect(wrapper.state().filterKey).toEqual("State")
    })

    // Test 20: Check sort and search
    it("Search and sort update correctly", () =>{
        const mount = createMount({dive: true})
        const wrapper = mount(<Search {...searchProps}/>)
        wrapper.find("input").at(0).simulate("change", {target: {value: "Joe"} })
        wrapper.find("select").at(0).simulate("change", {target: {value: "Name"} })
        expect(wrapper.state().searchValue).toEqual("Joe")
        expect(wrapper.state().sortValue).toEqual("Name")
    })
})