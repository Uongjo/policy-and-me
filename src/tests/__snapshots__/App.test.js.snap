// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components did load normally Bill loads normally 1`] = `
<div
  className="bill"
>
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
  >
    <h2>
      Bill-1
    </h2>
    <ul>
      <li>
        Introduced Date: 
        1999
      </li>
      <li>
        Primary Subject: 
        Saftey
      </li>
      <li>
        Sponsor: 
        Joe
      </li>
      <li>
        Committees: 
        ABC
      </li>
    </ul>
    
  </div>
</div>
`;

exports[`Components did load normally InterestGroup loads normally 1`] = `
<div
  className="interest-group"
>
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
  >
    <h2>
        
      Test
    </h2>
    <ul>
      <li>
        Address: 
        123 Test Ave
      </li>
      <li>
        City: 
        Somewhere
      </li>
      <li>
        Phone: 
      </li>
      <li>
        Email: 
        test@google.com
      </li>
    </ul>
    
  </div>
</div>
`;

exports[`Components did load normally Legislator loads normally 1`] = `
<div
  className="legislator"
>
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
  >
    <h2
      align="center"
    >
      First Last
    </h2>
    <div
      align="center"
    >
      <img
        alt="No picture"
        src="https://static.votesmart.org/canphoto/undefined.jpg"
      />
    </div>
    <ul>
      <li>
        Title: 
        1st Class
      </li>
      <li>
        Party: 
        D
      </li>
      <li>
        State: 
        TX
      </li>
      <li>
        Gender: 
        M
      </li>
    </ul>
    
  </div>
</div>
`;

exports[`Components did load normally NavBar loads normally 1`] = `
<div
  id="nav-bar"
>
  <div
    className="inner-nav-bar"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        className="logo"
        src="logo.png"
      />
    </a>
    <ul>
      <li>
        <a
          href="/about"
          onClick={[Function]}
        >
          ABOUT
        </a>
      </li>
      <li>
        <a
          href="/bills"
          onClick={[Function]}
        >
          BILLS
        </a>
      </li>
      <li>
        <a
          href="/legislators"
          onClick={[Function]}
        >
          LEGISLATORS
        </a>
      </li>
      <li>
        <a
          href="/interest-groups"
          onClick={[Function]}
        >
          INTEREST GROUPS
        </a>
      </li>
      <li>
        <a
          href="/visualizations"
          onClick={[Function]}
        >
          VISUALIZATIONS
        </a>
      </li>
    </ul>
  </div>
</div>
`;

exports[`Components did load normally Pagination loads normally 1`] = `
<div
  className="pagination"
>
  <button
    onClick={[Function]}
  >
    1
  </button>
  <button
    onClick={[Function]}
  >
    2
  </button>
  <button
    onClick={[Function]}
  >
    3
  </button>
  <button
    onClick={[Function]}
  >
    4
  </button>
  <button
    onClick={[Function]}
  >
    5
  </button>
  <button
    onClick={[Function]}
  >
    6
  </button>
  <button
    onClick={[Function]}
  >
    7
  </button>
  <button
    onClick={[Function]}
  >
    8
  </button>
  <button
    onClick={[Function]}
  >
    9
  </button>
  <button
    onClick={[Function]}
  >
    10
  </button>
</div>
`;

exports[`Components did load normally Person loads normally 1`] = `
<div
  className="person"
>
  <h3 />
  <div
    className="headshot-wrapper"
  >
    <img
      src="https://cdn130.picsart.com/305712853176201.jpg?c256x256"
    />
  </div>
  <p />
  <ul>
    <li>
      Responsibility: 
    </li>
    <li>
      Commits: 
    </li>
    <li>
      Issues: 
    </li>
    <li>
      Unit tests: 
    </li>
  </ul>
</div>
`;

exports[`Search, Sort, Filter Highlight did load normally 1`] = `
<highlightSearch
  instance={
    Object {
      "last": "clinton",
      "name": "bill",
      "state": "TX",
    }
  }
  searchValue="bill"
/>
`;

exports[`Search, Sort, Filter Search loads normally 1`] = `
<div
  className="search"
>
  <form>
    <div
      className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
        htmlFor="standard-basic"
      >
        Search
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          id="standard-basic"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
        />
      </div>
    </div>
  </form>
  <div
    className="MuiFormControl-root sort"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
      data-shrink={false}
      htmlFor="age-native-simple"
    >
      Sort By
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <select
        aria-invalid={false}
        className="MuiSelect-root MuiSelect-select MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
        disabled={false}
        multiple={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        value=""
      >
        <option
          value=""
        />
        <option
          value="name"
        >
          Name
        </option>
      </select>
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root MuiSelect-icon"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
    </div>
  </div>
  <br />
  <div
    className="MuiFormControl-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
      data-shrink={false}
      htmlFor="age-native-simple"
    >
      Filter By
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <select
        aria-invalid={false}
        className="MuiSelect-root MuiSelect-select MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
        disabled={false}
        multiple={false}
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        value=""
      >
        <option
          value=""
        />
        <option
          value="state"
        >
          State
        </option>
      </select>
      <svg
        aria-hidden="true"
        className="MuiSvgIcon-root MuiSelect-icon"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M7 10l5 5 5-5z"
        />
      </svg>
    </div>
    <div
      className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
        htmlFor="standard-basic"
      >
        Filter Value
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          id="standard-basic"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
        />
      </div>
    </div>
  </div>
  <br />
  <button
    onClick={[Function]}
  >
    Search
  </button>
</div>
`;

exports[`Search, Sort, Filter Splash did load normally 1`] = `
<div
  id="splash"
>
  <h1>
    POLICY AND ME
  </h1>
  <h3>
    Where policy and people meet
  </h3>
  <form>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline splash-search"
      onClick={[Function]}
    >
      <input
        aria-label="description"
        className="MuiInputBase-input MuiInput-input"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        placeholder="Search"
        type="text"
      />
    </div>
  </form>
  <br />
  <a
    href="/search/"
    onClick={[Function]}
  >
    <button>
      Search
    </button>
  </a>
</div>
`;

exports[`pages did load normally About did load normally 1`] = `
<div
  id="about"
>
  <div
    className="members"
  >
    <div
      className="person"
    >
      <h3>
        Abhishek Dayal
      </h3>
      <div
        className="headshot-wrapper"
      >
        <img
          src="abhishek_headshot.jpg"
        />
      </div>
      <p>
        Hi, my name is Abhishek! I'm from Dallas and my favorite food is the quesarito from Taco Bell.
      </p>
      <ul>
        <li>
          Responsibility: 
          AWS Connection, Database
        </li>
        <li>
          Commits: 
        </li>
        <li>
          Issues: 
        </li>
        <li>
          Unit tests: 
          10
        </li>
      </ul>
    </div>
    <div
      className="person"
    >
      <h3>
        Alec Sweet
      </h3>
      <div
        className="headshot-wrapper"
      >
        <img
          src="alec_headshot.png"
        />
      </div>
      <p>
        Hello! My name is Alec and I enjoy long walks on the beach.
      </p>
      <ul>
        <li>
          Responsibility: 
          Back-End, Database
        </li>
        <li>
          Commits: 
        </li>
        <li>
          Issues: 
        </li>
        <li>
          Unit tests: 
          30
        </li>
      </ul>
    </div>
    <div
      className="person"
    >
      <h3>
        Atharva Pendse
      </h3>
      <div
        className="headshot-wrapper"
      >
        <img
          src="atharva_headshot.jpg"
        />
      </div>
      <p>
        I’m a CS junior with an interest in theory and quant finance. When I’m free I like playing pool and listening to music.
      </p>
      <ul>
        <li>
          Responsibility: 
          AWS setup(Domain/SSL), Database
        </li>
        <li>
          Commits: 
        </li>
        <li>
          Issues: 
        </li>
        <li>
          Unit tests: 
          0
        </li>
      </ul>
    </div>
    <div
      className="person"
    >
      <h3>
        Joel Uong
      </h3>
      <div
        className="headshot-wrapper"
      >
        <img
          src="joel_headshot.PNG"
        />
      </div>
      <p>
        Hey! I'm Joel, a third year CS major, and I enjoy hiking.
      </p>
      <ul>
        <li>
          Responsibility: 
          Front-End
        </li>
        <li>
          Commits: 
        </li>
        <li>
          Issues: 
        </li>
        <li>
          Unit tests: 
          0
        </li>
      </ul>
    </div>
    <div
      className="person"
    >
      <h3>
        Malcolm Hess
      </h3>
      <div
        className="headshot-wrapper"
      >
        <img
          src="malcolm_headshot.PNG"
        />
      </div>
      <p>
        Hello, I am a senior computer science major with an interest in front end development, and I like to go bouldering
      </p>
      <ul>
        <li>
          Responsibility: 
          Full Stack
        </li>
        <li>
          Commits: 
        </li>
        <li>
          Issues: 
        </li>
        <li>
          Unit tests: 
          46
        </li>
      </ul>
    </div>
  </div>
  <div
    className="info"
  >
    <h1>
      Stats
    </h1>
    <p>
      <b>
        Total commits:
      </b>
       
      0
    </p>
    <p>
      <b>
        Total issues:
      </b>
       
      0
    </p>
    <p>
      <b>
        Total unit tests:
      </b>
       60
    </p>
    <p>
      <a
        href="https://documenter.getpostman.com/view/8970979/SVzua1tY"
        target="_blank"
      >
        API documentation
      </a>
    </p>
    <h1>
      Tools Used
    </h1>
    <p>
      <b>
        Front End:
      </b>
       React, Sass
    </p>
    <p>
      <b>
        Back End:
      </b>
       AWS, Flask
    </p>
    <p>
      <b>
        Other:
      </b>
       Postman, GitLab
    </p>
  </div>
</div>
`;

exports[`pages did load normally Bills did load normally 1`] = `
<div
  id="bills"
>
  <div
    className="search"
  >
    <form>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
          htmlFor="standard-basic"
        >
          Search
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            id="standard-basic"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
          />
        </div>
      </div>
    </form>
    <div
      className="MuiFormControl-root sort"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
        htmlFor="age-native-simple"
      >
        Sort By
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <select
          aria-invalid={false}
          className="MuiSelect-root MuiSelect-select MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
          disabled={false}
          multiple={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          value=""
        >
          <option
            value=""
          />
          <option
            value="short_title"
          >
            Title
          </option>
          <option
            value="sponsor_name"
          >
            Sponsor Name
          </option>
        </select>
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root MuiSelect-icon"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
    </div>
    <br />
    <div
      className="MuiFormControl-root"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
        htmlFor="age-native-simple"
      >
        Filter By
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <select
          aria-invalid={false}
          className="MuiSelect-root MuiSelect-select MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
          disabled={false}
          multiple={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          value=""
        >
          <option
            value=""
          />
          <option
            value="sponsor_party"
          >
            Sponsor Party
          </option>
          <option
            value="sponsor_state"
          >
            Sponsor State
          </option>
        </select>
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root MuiSelect-icon"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
          htmlFor="standard-basic"
        >
          Filter Value
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            id="standard-basic"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
          />
        </div>
      </div>
    </div>
    <br />
    <button
      onClick={[Function]}
    >
      Search
    </button>
  </div>
  <div
    className="pagination"
  />
  <div
    className="bill-grid"
  />
  <div
    className="pagination"
  />
</div>
`;

exports[`pages did load normally InterestGroups did load normally 1`] = `
<div
  id="interest-groups"
>
  <div
    className="search"
  >
    <form>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
          htmlFor="standard-basic"
        >
          Search
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            id="standard-basic"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
          />
        </div>
      </div>
    </form>
    <div
      className="MuiFormControl-root sort"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
        htmlFor="age-native-simple"
      >
        Sort By
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <select
          aria-invalid={false}
          className="MuiSelect-root MuiSelect-select MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
          disabled={false}
          multiple={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          value=""
        >
          <option
            value=""
          />
          <option
            value="name"
          >
            Name
          </option>
        </select>
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root MuiSelect-icon"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
    </div>
    <br />
    <div
      className="MuiFormControl-root"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
        htmlFor="age-native-simple"
      >
        Filter By
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <select
          aria-invalid={false}
          className="MuiSelect-root MuiSelect-select MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
          disabled={false}
          multiple={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          value=""
        >
          <option
            value=""
          />
          <option
            value="state"
          >
            State
          </option>
        </select>
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root MuiSelect-icon"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
          htmlFor="standard-basic"
        >
          Filter Value
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            id="standard-basic"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
          />
        </div>
      </div>
    </div>
    <br />
    <button
      onClick={[Function]}
    >
      Search
    </button>
  </div>
  <div
    className="pagination"
  />
  <div
    className="interest-groups-grid"
  />
  <div
    className="pagination"
  />
</div>
`;

exports[`pages did load normally Legislators did load normally 1`] = `
<div
  id="legislators"
>
  <div
    className="search"
  >
    <form>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
          htmlFor="standard-basic"
        >
          Search
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            id="standard-basic"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
          />
        </div>
      </div>
    </form>
    <div
      className="MuiFormControl-root sort"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
        htmlFor="age-native-simple"
      >
        Sort By
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <select
          aria-invalid={false}
          className="MuiSelect-root MuiSelect-select MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
          disabled={false}
          multiple={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          value=""
        >
          <option
            value=""
          />
          <option
            value="first_name"
          >
            First Name
          </option>
          <option
            value="last_name"
          >
            Last Name
          </option>
        </select>
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root MuiSelect-icon"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
    </div>
    <br />
    <div
      className="MuiFormControl-root"
    >
      <label
        className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
        data-shrink={false}
        htmlFor="age-native-simple"
      >
        Filter By
      </label>
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <select
          aria-invalid={false}
          className="MuiSelect-root MuiSelect-select MuiInputBase-input MuiInput-input MuiInputBase-inputSelect"
          disabled={false}
          multiple={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          value=""
        >
          <option
            value=""
          />
          <option
            value="party"
          >
            Party
          </option>
          <option
            value="state"
          >
            State
          </option>
        </select>
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root MuiSelect-icon"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>
      </div>
      <div
        className="MuiFormControl-root MuiTextField-root MuiFormControl-marginNormal"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated"
          data-shrink={false}
          htmlFor="standard-basic"
        >
          Filter Value
        </label>
        <div
          className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            className="MuiInputBase-input MuiInput-input"
            disabled={false}
            id="standard-basic"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
          />
        </div>
      </div>
    </div>
    <br />
    <button
      onClick={[Function]}
    >
      Search
    </button>
  </div>
  <div
    className="pagination"
  />
  <div
    className="legislator-grid"
  />
  <div
    className="pagination"
  />
</div>
`;
